apps/web/src/pages/Profile/LanguageInfoPage/components/LanguageInformationForm/LanguageInformationForm.tsx

```diff
@@ -236,6 +244,14 @@ const LanguageInformationForm: React.FunctionComponent<{
       ]}
       prefixBreadcrumbs={!application}
     >
+      {missingLanguageRequirements.length > 0 && (
```

Comment:
When checking length, we need to use a full ternary or else this will output 0 if the array is empty.

Reply:
Yeah, it does work. It is just safer to avoid using numbers on the left side and always use the full ternary when checking numbers. 😆 

Also, I recommend the [beta docs](https://beta.reactjs.org/learn/conditional-rendering#logical-and-operator-), so much better!

Reply:
OK, I'll avoid using the && operator unless it's an obvious boolean variable like an `isFoo` or a `hasFoo`.  I've switched it in [Switch to ternary conditional](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5557/commits/96f36d12492340a5a201201ec3383f8ccf3240dd).

Reply:
Thanks for the tip about the docs!  I didn't know about that.